docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+name: vita-min
+
+services:
+  db:
+    container_name: db
+    image: postgis/postgis:13-3.4-alpine
+    user: postgres
+    ports:
+      - 5432:5432
+    volumes:
+      - database:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+  jobs:
+    container_name: jobs
+    volumes:
+      - ./:/app
+    build:
+      dockerfile: Dockerfile.local
+      target: "base"
+    command: rails jobs:work
+    depends_on:
+      rails:
+        condition: service_healthy
+    environment:
+      - DOCKER=true
+      - RAILS_DB_HOST=db
+      - RAILS_DB_USERNAME=postgres
+      - RAILS_DB_PORT=5432
+      - RAILS_DB_PASSWORD=password
+  rails:
+    container_name: rails
+    build:
+      dockerfile: Dockerfile.local
+      target: "app"
+    volumes:
+      - ./:/app
+    ports:
+      - 3000:3000
+      - 9888:9888
+    depends_on:
+      - db
+      - shakapacker
+    links:
+      - db
+      - shakapacker
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://0.0.0.0:3000/up" ]
+      interval: 30s
+      timeout: 30s
+      retries: 15
+    environment:
+      - DOCKER=true
+      - RAILS_DB_HOST=db
+      - RAILS_DB_USERNAME=postgres
+      - RAILS_DB_PORT=5432
+      - RAILS_DB_PASSWORD=password
+      - SHAKAPACKER_DEV_SERVER_HOST=shakapacker
+  shakapacker:
+    container_name: shakapacker
+    build:
+      dockerfile: Dockerfile.local
+      target: "base"
+    volumes:
+      - ./:/app
+    ports:
+      - 3035:3035
+    command: "bin/shakapacker-dev-server --client-web-socket-url auto://0.0.0.0:3035/ws"
```

Comment:
Cool to see that this is in and working! Out of curiosity, what is the default url? Does it not work with docker compose?

Reply:
Not sure but it wasn't working and I didn't want to change the config and risk disrupting local install, so using this flag seemed the safest bet for now.